homeassistant/components/huawei_lte/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
```

Comment:
Let's have `test_connection_errors` finish in CREATE_ENTRY. This way we also test that the flow is able to recover from an error

Reply:
Idem for `test_login_error`

Reply:
Noted in 6f4b51d1eebeb8405feeff141d01bde7000425b1